galette/templates/default/pages/members_list.html.twig

```diff
@@ -269,8 +269,12 @@
                         <tr>
                             <th><i class="ui user shield red icon"></i></th>
                             <td class="back">{{ _T('Admin') }}</td>
-                            <th><i class="ui id card alternate orange icon"></i></th>
+                            <th><i class="ui id card alternate red icon"></i></th>
```

Comment:
Why "red"? See https://github.com/galette/galette/pull/308/files#diff-6a061e5c740dc94251e5ef336699fd7126a7fd5e40ded2bb7764d048f7f8d95fR339

red for admins, orange for group manager, andn orange (or another color) for groups members seems correct.

```suggestion
                            <th><i class="ui id card alternate orange icon"></i></th>
```

Reply:
BTW, this currently does not work... `isStaff()` always return false on members list due to how members are loaded (I guess this is another "old" bug).